src/core/schema.ts

```diff
@@ -29,6 +30,22 @@ export const defaultSchema: Schema = {
     // [0-9]
     ...objectFromEntries("0123456789".split("").map((n) => [n, n])),
   },
+  defaultEventNames: {
+    a: () => "click",
+    button: () => "click",
+    form: () => "submit",
+    details: () => "toggle",
+    input: (e) => (e.getAttribute("type") == "submit" ? "click" : "input"),
```

Comment:
```suggestion
    input: (element) => (element.getAttribute("type") == "submit" ? "click" : "input"),
```

Flyby comment - I know this is copied from the original code but maybe we can make this more readable by using a nicer variable name here.